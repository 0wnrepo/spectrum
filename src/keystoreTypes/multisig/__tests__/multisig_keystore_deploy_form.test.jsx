import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import MultisigKeystoreDeployForm from '../multisig_keystore_deploy_form';

describe('<MultisigKeystoreDeployForm />', () => {
  test('renders correctly', () => {
    const component = shallow(
      <MultisigKeystoreDeployForm
        formData={{}}
        formChange={jest.fn}
      />,
    );

    expect(toJson(component)).toMatchSnapshot();
  });
});
