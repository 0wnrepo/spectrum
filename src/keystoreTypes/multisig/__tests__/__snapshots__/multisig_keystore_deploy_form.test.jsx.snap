// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MultisigKeystoreDeployForm /> renders correctly 1`] = `
<FormField>
  <FormField>
    <label
      htmlFor="from"
    >
      From
    </label>
    <Connect(DefaultAddressSelector) />
  </FormField>
  <FormGroup
    widths="equal"
  >
    <FormField
      formChange={[Function]}
      formData={Object {}}
      label="Number of Initial Onwers"
      max={10}
      min={1}
      name="count"
      step={1}
      type="number"
    />
    <FormField
      formChange={[Function]}
      formData={Object {}}
      label="Reqruied Quorum for Executions"
      max={10}
      min={1}
      name="required"
      step={1}
      type="number"
    />
  </FormGroup>
  <FormField>
    <label
      htmlFor="owners"
    >
      Owners
    </label>
  </FormField>
</FormField>
`;
